.github/workflows/e2e_tests.yml

```diff
@@ -0,0 +1,211 @@
+name: e2e-tests
+on:
+  push:
+    tags:
+    - '*tetrate*'
+  
+  workflow_dispatch:
+
+jobs:
+  eks-e2e-test:
+    if: false
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+
+    name: eks-e2e-test
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: aws-config
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_REGION  }}
+
+      - name: set up go 1.15
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.15  
+
+      - name: eks-cluster-create
+        run: bash ./tetrateci/create_eks_cluster.sh
+
+      - name: operator-tests
+        run: go test -tags=integ ./tests/integration/operator/...  -istio.test.skipVM true -p 1  -test.v
+      
+      - name: helm-tests
+        run: go test -tags=integ ./tests/integration/helm/... -istio.test.skipVM true -p 1 -test.v
+
+      - name: pilot-tests
+        run: |
+          go test -tags=integ -timeout 30m -run='TestEmptyCluster|TestFileOnly|TestDirectoryWithoutRecursion|TestDirectoryWithRecursion|TestInvalidFileError|TestJsonInputFile|TestJsonOutput|TestKubeOnly|TestFileAndKubeCombined|TestAllNamespaces|TestTimeout|TestErrorLine|TestWait|TestVersion|TestDescribe|TestAddToAndRemoveFromMesh|TestProxyConfig|TestProxyStatus|TestAuthZCheck|TestLocality|TestMain|TestMirroring|TestMirroringExternalService|TestTproxy|TestRevisionedUpgrade|TestValidation|TestEnsureNoMissingCRDs|TestWebhook' ./tests/integration/pilot/ -istio.test.skipVM true -p 1 -test.v 
```

Comment:
Instead can we exclude the known test cases that fails? That way any new tests added would be taken up

Reply:
I tried that at first but go's regex engine doesn't allow not operator.

Reply:
Please create an issue - we need to address this.